config/settings.yml

```diff
@@ -227,6 +227,9 @@ default_projects_modules:
   - time_tracking
   - timelines
   - wiki
+  - meetings
+  - reporting_module
+  - backlogs
```

Comment:
@ulferts then I please ask you to take care about this. @jonasheinrich started with this but testing his PRs did not work for me. you may check them as well.